src/app.ts

```diff
@@ -9,11 +9,16 @@ import * as fs from 'fs';
 import * as http from 'http';
 import * as https from 'https';
 const debug = Debug('universalmediaserver-api:server');
+import * as koaPrometheus from 'koa-prometheus-adv';
+import * as client from 'prom-client';
 import indexRouter from './routes/index';
 import mediaRouter  from './routes/media';
 import { ExternalAPIError, IMDbIDNotFoundError, MediaNotFoundError, ValidationError } from './helpers/customErrors';
 
-const app = new Koa();
+client.collectDefaultMetrics({ register: client.register });
+// @ts-ignore
```

Comment:
Any ideas on this one, if you get the chance to look?

Reply:
The koa-prometheus dependency is designed to work with prometheus v11 but we use v14, I think that's the reason